scripts/apps/ingest/directives/IngestSourcesContent.ts

```diff
@@ -580,6 +581,14 @@ export function IngestSourcesContent(ingestSources, notify, api, $location,
                     return feedingService ? feedingService.templateUrl : '';
                 };
 
+                /**
+                 * Do URL request specified in url_request field
+                 * @param {Object} field
+                 */
+                $scope.doUrlRequest = (field) => {
+                    $window.open(field.url);
```

Comment:
why not use the native `window`? Is the angular version doing anything special?

Reply:
I guess it should work too, I've just checked how people are usually doing with Angular.